### ISSUE TITLE
Nonexistence of node_modules makes wrong paths

### ISSUE THREAD

**Issue by Cactucs** [medikoo/modules-webmake#54]
See: https://github.com/manGoweb/mango-cli/issues/98

```
correct:
www/src/scripts/index

wrong:
/ekola/www/src/scripts/index

project root:
W:/rekola/www
```

**Comment by medikoo** [comment]
@Cactucs Webmake detects root of a package, either by existance of top most `node_modules` folder or by existence of top most `package.json` file.  If neither do not exist then root of the system will be assumed.

This behavior while may seem controversial, was intended (still it will be optimized with next major version). For now I would suggest to put `package.json` in root of a project, it will not do harm and it will indicate that we deal with root of node.js project.

The only issue I see is that path resolves to `/ekola/www/src/scripts/index` and not `/rekola/www/src/scripts/index` on your side (assuming root is `W:/rekola/www`). Is it indeed the case, or it's a typo on your side?

**Comment by Cactucs** [comment]
Is it posible to pass root path (we "replacement" of package.json file)?

**Comment by medikoo** [comment]
> Is it posible to pass root path (we have "replacement" file for package.json mango.json)?

No, it follows npm and node.js rules, and that handling is deeply hardcoded.
So at this point the only options are:
- You're fine with path going down to root
- You provide either `package.json` file or `node_modules` folder to ensure proper detection of node project root
- Wait for next major version (it'll not compute _root_ not on basis of existence of `package.json` or `node_modules`, but on basis of top most bundled module).

Let me leave this issue open until we have other nicer _project's root_ calculation
